### PR TITLE
Fixed config bug

### PR DESCRIPTION
**Pull request by Ariamiro**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -163,11 +163,19 @@ io.on('connection', function (socket) {
     });
 
     socket.on('disconnect', function () {
-        var playerIndex = findIndex(users, userID);
-        var playerName = users[playerIndex].name;
-        users.splice(playerIndex, 1);
+        var playerName = findPlayer(userID).name;
```
Comment (RemoveIt) [inline server/server.js]: what does this have to config file?

Comment (Ariamiro) [inline server/server.js]: nothing, accidental.

Comment (Ariamiro) [inline server/server.js]: no, but is part of repo

**Comment by Ariamiro** [comment]
don't need anymore

**Pull request closed without merging**